diff -a -p -u -r1.8 cui.cpp
--- cui.cpp 19 Dec 2006 20:35:33 -0000 1.8
+++ cui.cpp 5 May 2008 07:40:02 -0000
@@ -1,5 +1,6 @@
/* NetHogs console UI */
#include <string>
+#include <cstdlib>
#include <pwd.h>
#include <sys/types.h>

diff -a -p -u -r1.5 inodeproc.cpp
--- inodeproc.cpp 19 Dec 2006 20:35:33 -0000 1.5
+++ inodeproc.cpp 5 May 2008 07:40:02 -0000
@@ -1,6 +1,7 @@
#include <sys/types.h>
#include <errno.h>
#include <string.h>
+#include <cstdlib>
#include <dirent.h>
#include <ctype.h>
#include <iostream>
